certbot-ci/certbot_integration_tests/certbot_tests/test_main.py

```diff
@@ -98,3 +98,33 @@ def test_renew_with_hook_scripts(context):
 
     assert_cert_count_for_lineage(context.config_dir, certname, 2)
     assert_hook_execution(context.hook_probe, 'deploy')
+
+
+def test_ocsp_status(context):
+    """Test retrieval of OCSP statuses."""
+    if context.acme_server == 'pebble':
+        pytest.skip('Pebble does not support OCSP status requests.')
```

Comment:
Can we either:

1. Move this skip after checking the stale OCSP status.
2. Split this into two functions and only skip the code under "OCSP 2".

Reply:
Solution 2. Otherwise the whole test would be marked as skipped even if one part as been executed.

Reply:
Implemented.